server/api/classyear/classyear.controller.js

```diff
@@ -27,6 +27,20 @@ exports.getClassYear = function(req, res) {
   })
 };
 
+// Get a specific class year's attendance bonus days
+exports.countBonusDays = function(req, res) {
+  ClassYear.findOne({
+    "current": true
+  }, function (err, classYear){
+    if(err) { return handleError(res, err); }
+    if (!classYear) return res.send(404);
+    res.json(classYear.dayCodes.reduce(function(previousValue, currentValue, index, array) {
```

Comment:
might be nice to break this into multiple lines and comment on what it is doing with the reduce. People new to js will have a hard time understanding this